docs/user/custom-resources/README.md

```diff
@@ -0,0 +1,21 @@
+# Custom resources
+
+## APIRule custom resource <!-- {docsify-ignore} -->
+
+The `apirules.gateway.kyma-project.io` CustomResourceDefinition (CRD) describes the kind and the format of data the APIRule Controller uses to configure the resources.
+
+Browse the documentation related to the APIRule custom resource (CR):
```

Comment:
The following links for APIRule CR are broken.

Reply:
thanks, fixed